test/integration/http2_integration_test.cc

```diff
@@ -221,7 +221,7 @@ TEST_P(Http2IntegrationTest, IdleTimeoutWithSimultaneousRequests) {
     auto* cluster = static_resources->mutable_clusters(0);
     auto* http_protocol_options = cluster->mutable_common_http_protocol_options();
     auto* idle_time_out = http_protocol_options->mutable_idle_timeout();
-    std::chrono::milliseconds timeout(1000);
+    std::chrono::milliseconds timeout(3000);
```

Comment:
Why the changes here? Is there some other underlying timing issue with the test?

Reply:
@mattklein123 the test failed only on Mac once with not getting response for second request. I suspected that it could be because of idletimeout and increased it. But looking at the in-flight PRs it seems to be working fine. So I will revert this and we can revisit if it causes problem again.